modelcards/card_data.py

```diff
@@ -73,31 +73,72 @@ class EvalResult:
 
 @dataclass
 class CardData:
-    """Model Card Metadata that is used by Hugging Face Hub when included at the top of your README.md
+    def __init__(
+        self,
+        language: Optional[Union[str, List[str]]] = None,
+        license: Optional[str] = None,
+        library_name: Optional[str] = None,
+        tags: Optional[List[str]] = None,
+        datasets: Optional[Union[str, List[str]]] = None,
+        metrics: Optional[Union[str, List[str]]] = None,
+        eval_results: Optional[List[EvalResult]] = None,
+        model_name: Optional[str] = None,
+        **kwargs,
+    ):
+        """Model Card Metadata that is used by Hugging Face Hub when included at the top of your README.md
+
+        Args:
+            language (Optional[Union[str, List[str]]], optional):
```

Comment:
I wonder if we need the extra `optional` if you're putting the complete type here which already includes `Optional`.

Reply:
I would put the following:

```suggestion
            language (`Union[str, List[str]]`, *optional*):
```

Reply:
Cool, thanks Lysandre!! @adrinjalali I'm going to address this in a separate PR when I go to resolve #19 - that way I don't have to do this twice right now.

Reply:
Sure